docs/dust_attenuation/fit_attenuation.rst

```diff
@@ -15,45 +15,145 @@ In this example, we create some artificial curve with the C00 model
 and explain how to fit an attenuation curve with it.
 
 .. plot::
-   :include-source:
+    :include-source:
 
-   import matplotlib.pyplot as plt
-   import numpy as np
+    import matplotlib.pyplot as plt
+    import numpy as np
 
-   from astropy.modeling.fitting import LevMarLSQFitter
+    from astropy.modeling.fitting import LevMarLSQFitter
+    import astropy.units as u
 
-   from dust_extinction.dust_extinction import C00
-   
-   # Create artificial attenuation curve with C00 and Av = 1.3 mag
-   x = np.arange(0.5,8.0,0.1)/u.micron
-   att_model = C00(Av=1.3)
-   y = att_model(x)
-   # add some noise 
-   noise = np.random.normal(0, 0.2, y.shape)
-   y+=noise
+    from dust_attenuation.C00 import C00
 
-   # initialize the model
-   c00_init = C00()
+    # Create artificial attenuation curve with C00 and Av = 1.3 mag
+    # Better sampling using wavenumbers
+    x = np.arange(0.5,8.0,0.1)/u.micron
+    # Convert to microns
+    x=1/x
 
-   # pick the fitter
-   fit = LevMarLSQFitter()
+    att_model = C00(Av=1.3)
+    y = att_model(x)
+    # add some noise
+    noise = np.random.normal(0, 0.2, y.shape)
+    y+=noise
 
-   # fit the data to the FM90 model using the fitter
-   #   use the initialized model as the starting point
-   c00_fit = fit(c00_init, x.value, y)
+    # initialize the model
+    c00_init = C00()
 
-   # plot the observed data, initial guess, and final fit
-   fig, ax = plt.subplots()
+    # pick the fitter
+    fit = LevMarLSQFitter()
 
-   ax.plot(x, y, 'ko', label='Observed Curve')
-   ax.plot(x.value, c00_init(x.value), label='Initial guess')
-   ax.plot(x.value, c00_fit(x.value), label='Fitted model')
+    # fit the data to the FM90 model using the fitter
+    #   use the initialized model as the starting point
+    c00_fit = fit(c00_init, x.value, y)
 
-   ax.set_xlabel('$x$ [$\mu m^{-1}$]')
-   ax.set_ylabel('$Ax $')
+    print ('Fit results:\n', c00_fit)
+    # plot the observed data, initial guess, and final fit
+    fig, ax = plt.subplots()
 
-   ax.set_title('Example C00 Fit ')
+    ax.plot(1/x, y, 'ko', label='Observed Curve')
+    ax.plot(1/x.value, c00_init(x.value), label='Initial guess')
+    ax.plot(1/x.value, c00_fit(x.value), label='Fitted model')
 
-   ax.legend(loc='best')
-   plt.tight_layout()
-   plt.show()
+    ax.set_xlabel('$x$ [$\mu m^{-1}$]')
+    ax.set_ylabel('$Ax $')
+
+    ax.set_title('Example C00 Fit ')
+
+    ax.legend(loc='best')
+    plt.tight_layout()
+    plt.show()
+
+
+
+Example: Use WG00 to fit C00
+=================
```

Comment:
This line is too short.  The number of characters in the "====..." line has to be the same as the text above it.

Reply:
This issue happens two other times and is the cause of the readthedocs error.